### ISSUE TITLE
release containing ruby versions 2.2.8, 2.3.5, 2.4.2

### ISSUE THREAD

**Issue by josephholsten** [rvm/rvm#4168]
There are ruby security fixes in the rubies in https://github.com/rvm/rvm/pull/4159, I'd love a release with those as soon as possible.

It would be great if we could prioritize making a release immediately after patches that affect security. I don't know what I can do to help with that, but I'm happy to help in any way!

**Comment by mpapis** [comment]
We are fresh after a release, it's a time consuming process, try `rvm get master` - it's all there.

**Comment by josephholsten** [comment]
is there anything I can do to help make the release process less painful/time-consuming?

**Comment by mpapis** [comment]
There are two problems:

1. Reviewing recent issues and checking if there is nothing that would break something
2. Signing the releases

Most of the processes is automated, but unfortunately it's still consuming, and the more security and failover we are adding the bigger it will grow, I miss the times when I had to just push to master and it was all good. Now we got stable branch, git flow, two remotes, signing releases.
